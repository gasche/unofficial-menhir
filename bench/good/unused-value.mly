%token FOO BAR
%start <unit> main

%%

main:
  f = FOO b = BAR
    { () }

