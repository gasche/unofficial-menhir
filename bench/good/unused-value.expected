%start main
%token FOO
%token BAR

%type <unit> main
%%

main:
| f = FOO b = BAR
    { () }

%%



